### DISCUSSION TITLE
WebAuth Error

### DISCUSSION THREAD

**Discussion by metasoul111** [Web3Auth/Web3Auth#1149] [Help]
Some of our end users are getting the follow error when logging in. We are using the standard Web2Auth plug and play.

![Screenshot from 2023-01-02 13-23-29](https://user-images.githubusercontent.com/89988259/210272472-d2dbdbf7-f59d-4c96-bc4f-73e152d4a6a1.png)

**Comment by guru-web3** [comment]
can you share the web3auth code setup snippet if possible?

**Reply by metasoul111** [reply]
@guru-web3 here is the setup snippet

import OpenLogin from '@toruslabs/openlogin';

const openLogin = new OpenLogin({
    clientId: process.env.REACT_APP_TORUS_PROJECT_ID,
    network: process.env.REACT_APP_TORUS_NETWORK,
    uxMode: 'popup',
    whiteLabel: {
      name: 'META1'
    },
  });

const renderTorusStep = async () => {
    if (
      !openLogin
    ) {
      return;
    }
try {
  await openLogin.init();
  await openLogin.login();
  if (openLogin.privKey) {
    const privKey = openLogin.privKey;
    const data = await openLogin.getUserInfo();

    setAuthData(data);
    setPrivKey(privKey);
    setEmail(data?.email.toLowerCase());
    setLoader(false);
    console.log('User logged in');
    setStep('faceki');
  }
} catch (error) {
  console.log('Error in Torus Render', error);
  setLoader(false);
}

  }

**Reply by metasoul111** [reply]
"@toruslabs/openlogin": "^2.10.0",

**Reply by guru-web3** [reply]
please refer this example https://github.com/torusresearch/OpenLoginSdk/blob/master/examples/react-example/src/App.tsx and cross verify the Openlogin init, login methods,
make sure you are using correct network

**Reply by antmanstar** [reply]
![image](https://user-images.githubusercontent.com/43899645/211006121-bfa4aa30-726e-406b-9c41-1abdb3e45daa.png)

![image](https://user-images.githubusercontent.com/43899645/211006346-27fa9eb4-e379-4a22-a224-7ae1bce4ee26.png)

I've got the different error.
